project/make/validate-toml

```diff
@@ -9,7 +9,7 @@ unset IFS
 badFiles=()
 for f in "${files[@]}"; do
 	# we use "git show" here to validate that what's committed is formatted
-	if [ "$(git show "$VALIDATE_HEAD:$f" | tomlv)" ]; then
+	if [ "$(git show "$VALIDATE_HEAD:$f" | tomlv /proc/self/fd/0)" ]; then
```

Comment:
Actually, this should probably even be checking the return value of `tomlv` instead, ie:
```bash
if ! git show ...... | tomlv ... ]; then
```